src/randoop/JunitFileWriter.java

```diff
@@ -5,12 +5,11 @@
 import java.io.PrintStream;
 import java.util.ArrayList;
 import java.util.Collections;
+import java.util.Iterator;
 import java.util.LinkedHashMap;
 import java.util.LinkedList;
 import java.util.List;
 import java.util.Map;
-import java.util.Set;
-
```

Comment:
I added an Eclipse project specific setting for organize imports that recognizes randoop as a block, so running organize imports on a file will result in the randoop imports being separated from java ones.